app/js/handlebarsScoreTable.ts

```diff
@@ -339,6 +344,24 @@ async function addEventListenersForViewResults() {
         });
     });
 }
+/*
+* function that hide button answer and breakdown from view result if
+* the user has not taken the test aptitude v1
+ */
+function hideButton(){
```

Comment:
this function name is too generic. Hide what button? there are hundreds in the application

Reply:
also missing a space before the `{`